packages/basis/src/BasisLoader.ts

```diff
@@ -1,6 +1,6 @@
 import { TYPES, MIPMAP_MODES, ALPHA_MODES, FORMATS } from '@pixi/constants';
 import { BaseTexture, BufferResource, Texture } from '@pixi/core';
-import { CompressedTextureResource, INTERNAL_FORMATS } from '@pixi/compressed-textures';
+import { CompressedTextureResource, INTERNAL_FORMATS, CompressedLevelBuffer } from '@pixi/compressed-textures';
```

Comment:
`CompressedLevelBuffer` should probably be imported as a type (after all regular imports):

```ts
import type { CompressedLevelBuffer } from '@pixi/compressed-texture';
```